api/src/Common/Models/MemberMatchRecord.cs

```diff
@@ -1,13 +1,19 @@
+using System;
 using System.ComponentModel.DataAnnotations;
 
 namespace RaceResults.Common.Models
 {
-    public class MemberMatchRecord
+    public class MemberMatchRecord : IModel
     {
         [Required]
-        public string Id { get; set; }
+        public Guid Id { get; set; }
```

Comment:
Currently, this is the name of the record, not a guid. We should leave this a string for now.

Reply:
To compile, this class must implement IModel, which requires a Guid Id

Reply:
Well, yes but with your changes to the provider. It compiled fine before.

Reply:
No, it doesn't compile on main

Reply:
MemberMatch.csproj doesn't compile, I mean

Reply:
The problem is that this change will throw an exception if we get any records. Id needs to be a string

Reply:
I can back the changes out, but then MemberMatch.csproj won't compile again

Reply:
I backed this out